images/linux/post-generation/homebrew-permissions.sh

```diff
@@ -5,7 +5,7 @@
 brew_folder="/home/linuxbrew/"
 if [ -d "$brew_folder" ]; then
     brew_folder_owner=$(ls -ld $brew_folder | awk '{print $3}')
-    if [ "$USER" != "$brew_folder_owner" ]; then
-        chown "$USER":docker -R $brew_folder
+    if [ "$SUDO_USER" != "$brew_folder_owner" ]; then
```

Comment:
I am not sure and need to validate it. As I know we use runneradmin under provisioner process not the runner.